src/components/Metrics/DAOMetricsHeader.jsx

```diff
@@ -69,7 +69,7 @@ export default function DAOMetricsHeader({ metrics }) {
                       side="bottom"
                       sideOffset={3}
                     >
-                      <span>Total amount of OP in existence</span>
+                      <span>Total amount of {token.symbol} in existence</span>
                     </HoverCardContent>
```

Comment:
Just want to confirm that `.symbol` is declared for both op and ether.fi.  From line 63, I trust it must have been set for OP before(?), but I don't see a `symbol = ETHFI` in this PR.

Reply:
Correct, the token symbol is set for each tenant.